shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/EraMap.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | This module provides the mapping from conceptual rule names to concrete
+-- rules for the Shelley era.
+module Shelley.Spec.Ledger.STS.EraMap () where
```

Comment:
So this module will only contain the Shelley mapping. How should the modules containing the other mappings be called? `Cardano.Ledger.ShelleyMA.STS.EraMap` and/or `Cardano.Ledger.Allegra/Mary.STS.EraMap`, etc?

For consistency, what about calling this module `Cardano.Ledger.Shelley.STS.EraMap`?

Reply:
Minor: when I see the `-Map` in `EraMap` I think of `Map` the data type instead of the mapping. What about `EraMapping`?

Reply:
At some point I want to move the whole `Shelley` codebase to `Cardano.Ledger.Shelley`. And `STS` to `Rule`, indeed...

Reply:
Sorry I made a similar comment about this. So please ignore it.